examples/chart/teleport-kube-agent/README.md

```diff
@@ -53,6 +53,34 @@ Set the values in the above command as appropriate for your setup.
 You can also optionally set labels for your Kubernetes cluster using the
 format `--set "labels.key=value"` - for example: `--set "labels.env=development,labels.region=us-west-1"`
 
+It's possible to create secret with the auth token beforehand. To do so, run:
```

Comment:
```suggestion
To avoid specifying the auth token in plain text, it's possible to create a secret containing the token beforehand. To do so, run:
```